### PR TITLE
ADD: basic urdf model of fts150

### PR DESCRIPTION
**Pull request by lorenz-halt**
and a launch files to test it.
See #31

### CODE REVIEW

File: robotiq_force_torque_sensor/launch/ur5_fts_test.launch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+
+    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
+        <param name="use_gui" value="true" />
+    </node>
+
+    <!-- rviz -->
+    <node pkg="rviz" type="rviz" name="rviz"/>
+</launch>
```
Comment (gavanderhoorn) [inline robotiq_force_torque_sensor/launch/ur5_fts_test.launch:14]: Is this file necessary? It adds a dependency (through `ur5_fts_test.urdf.xacro`) `ur_description`.

Seems strange to make the `robotiq` pkg depend on an arbitrary robot vendor support pkg?

File: robotiq_force_torque_sensor/nodes/rq_sensor.cpp [hunk 2]
```diff
@@ -177,7 +177,7 @@ int main(int argc, char **argv)
 
 	//std_msgs::String msg;
 	geometry_msgs::WrenchStamped wrenchMsg;
-	ros::param::param<std::string>("~frame_id", wrenchMsg.header.frame_id, "robotiq_force_torque_frame_id");
+	ros::param::param<std::string>("~frame_id", wrenchMsg.header.frame_id, "fts_measurement_frame");
```
Comment (gavanderhoorn) [inline robotiq_force_torque_sensor/nodes/rq_sensor.cpp:180]: What motivated this change? I wasn't involved in the introduction of `robotiq_force_torque_frame_id`, so I don't know why that name was chosen, but this introduces a potentially breaking change without a rationale.

File: robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <collision>
+                <origin xyz="0 0 0.01875" rpy="0 0 0" />
+                <geometry>
+                    <cylinder length="0.0375" radius="0.060"/>
+                </geometry>
+            </collision>
+        </link>
+
+        <!-- virtual measurement frame -->
+        <joint name="${prefix}measurment_joint" type="fixed" >
```
Comment (gavanderhoorn) [inline robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro:38]: typo: `measurment` -> `measurement`.

File: robotiq_force_torque_sensor/urdf/ur5_fts_test.urdf.xacro [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+    <!-- include universal_robot arm -->
+    <xacro:ur5 prefix=""/>
+
+    <!-- include fts -->
+    <xacro:robotiq_fts150 prefix="" parent="ee_link">
+        <origin xyz="0.009 0 0" rpy="0 ${pi/2} 0"/> <!-- This offset is the flange adapter plate -->
+    </xacro:robotiq_fts150>
+
+</robot>
```
Comment (gavanderhoorn) [inline robotiq_force_torque_sensor/urdf/ur5_fts_test.urdf.xacro:16]: See earlier comment about dependency on `ur_description`.

A test / model inspection launch file is a nice addition, but perhaps it should not introduce dependencies on external packages from unrelated repositories.

File: robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```
Comment (gavanderhoorn) [inline robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro:1]: Are there any best practices for these kind of xacros? Link and joint names do not appear to be harmonised / follow some kind of convention.

File: robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="robotiq_fts150">
+
+    <xacro:macro name="robotiq_fts150" params="parent prefix *origin">
+
+        <!-- mount the fts to the robot -->
+        <joint name="{prefix}fts_fix" type="fixed" >
+            <!--In most cases origin would be flange adapter plate: <origin xyz="0 0 0.009" rpy="0 0 0"/> /-->
```
Comment (gavanderhoorn) [inline robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro:8]: Where does the (commented) `origin xyz="0 0 0.009"` come from?

**Comment by gavanderhoorn** [comment]
A bit after-the-fact, I know, but still.

File: robotiq_force_torque_sensor/nodes/rq_sensor.cpp (hunk 2 above)
Comment (shaun-edwards) [inline robotiq_force_torque_sensor/nodes/rq_sensor.cpp:180]: This change isn't backwards compatible.  Anyone using the `robotiq_force_torque_frame_id` or not setting the `~frame_id` will have problems when this is merged.

Was there a reason to change the frame name?

File: robotiq_force_torque_sensor/launch/ur5_fts_test.launch (hunk 1 above)
Comment (lorenz-halt) [inline robotiq_force_torque_sensor/launch/ur5_fts_test.launch:14]: You are right, its the testcase, i'll remove it

File: robotiq_force_torque_sensor/nodes/rq_sensor.cpp (hunk 2 above)
Comment (lorenz-halt) [inline robotiq_force_torque_sensor/nodes/rq_sensor.cpp:180]: there is really no reason for a name change, since I introduced this name I felt like the "new" name is more intuitive. But you are right, if some actually already adapted this a change is bad. So I'll keep the initial frame_id and change the urdf accordingly

File: robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro (hunk 6 above)
Comment (lorenz-halt) [inline robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro:8]: This is the width of the fts150 ur-kit adapter-plate
